students/jayjohnson/lesson02/Fizz_Buzz.py

```diff
@@ -0,0 +1,28 @@
+'''
+Jay Johnson Fizz Buzz
+
+x Write a program that prints the numbers from 1 to 100 inclusive.
+x But for multiples of three print “Fizz” instead of the number.
+x For the multiples of five print “Buzz” instead of the number.
+x For numbers which are multiples of both three and five print “FizzBuzz” instead.
+'''
+# loop logic
+guess = False
+
+# count set to 100
+test_num = 100
+
+# loop
+while guess == False:
```

Comment:
An interesting fizzbuzz! The logic works. 
A few things to think about:
- This goes in the opposite order from usual. How would you do it in the traditional 1-100 order?
- guess is False, but then you never reassign it, so it's always False and you just break from the loop with break. So you can take out the guess and just start with:
```
while True
```